app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+    def full_title page_title = ""
+        base_title = "Ruby on Rails Tutorial Sample App"
+        page_title.empty? ? base_title : "#{page_title} | #{base_title}"
```

Comment:
use I18n for ` "Ruby on Rails Tutorial Sample App"` 
link: https://guides.rubyonrails.org/i18n.html

Reply:
ok done

Reply:
please tab 2 space